src/js/components/Organization.js

```diff
@@ -113,6 +113,13 @@ export default class Organization extends Component {
 
   render () {
     const person = this.props.person;
+    let givenName = person.givenName;
+    // check to see if givenName includes single letter (middle initial)
+    // after first name, and trim it
+    if (givenName && givenName.indexOf(' ') === givenName.length - 2) {
```

Comment:
Can we use regular expression instead? It would be nice to support dot on the name as well. For example `R.`